File: aula11/sortear_sena.py
# sorteia 6 dezenas para a Sena (1 a 60) */

from random import randrange

NUM_DEZENAS = 6
dezenas = []
for i in range(NUM_DEZENAS):
    repetido = True
    while repetido: 
        sorteio = randrange(1, 61)
        if sorteio not in dezenas:
            repetido = False
    dezenas.append(sorteio)
for n, dezena in enumerate(dezenas, 1):
    print 'dezena',n,':',dezena

File: aula11/SortearSena.java
/* sorteia 6 dezenas para a Sena (1 a 60) */

public class SortearSena {
	static final int NUM_DEZENAS = 6;
	public static void main(String [] args) {
		int[] dezenas = new int[NUM_DEZENAS];
		int sorteio;
		boolean repetido;
		for (int i = 0; i < NUM_DEZENAS; i++) {
			do {
				sorteio = (int)(Math.random() * 60) + 1;
				repetido = false;
				for (int j = 0; j < i; j++) {
					if (sorteio == dezenas[j]) {
						repetido = true;
						break;
					}
				}					
			} while (repetido);
			dezenas[i] = sorteio;
		}
		for (int i = 0; i < NUM_DEZENAS; i++) {
			System.out.println("dezena " + (i+1) + ": " + dezenas[i]);
		}
	}
}
